jest.conf.json

```diff
@@ -2,6 +2,9 @@
   "roots": ["./src/app"],
   "testRegex": "/src/app/.*\\.test.(js|jsx)$",
   "unmockedModulePathPatterns": [
-      "./node_modules/react"
+    "./node_modules/react"
+  ],
+  "setupFiles": [
+    "./testSetup.js"
```

Comment:
please rename to `test-setup.js`


we use `CamelCase`  for component  names. one component per file, file name corresponds to component name (e.g. TextField.js)
all other files used `dash-case` naming.